app.js

```diff
@@ -227,3 +229,15 @@ closeImg.addEventListener('click', function (e) {
   html.classList.remove('modal-noscroll');
   document.location.reload();
 });
+
+// handel form validation
+submit.addEventListener('click', function () {
+  if (emailInput.value !== emailInput.value.toLowerCase()) {
+    resultMsg.textContent = 'Email must be in lower case';
+    resultMsg.style.color = '#000';
+    return false;
+  } else {
+    resultMsg.style.color = 'rgb(255,255,255)';
+    resultMsg.textContent = 'Valid email';
+  }
+});
```

Comment:
**[Mandatory]**

- [ ] We are still able to submit your form even with a capital letter in the email. Kindly prevent your form from being submitted if it has a capital letter in the email address 😊
- [ ] Your Error message comes underneath your input field. Kindly make your error near the submit button like direct below the submit button or direct above the submit button 👍 to follow the requirements

**[Optional]**

- [ ] Your error message color should be in red color cuz its a standard that error messages should be in red color 👍